managed/CounterStrikeSharp.API/Modules/Menu/CenterHtmlMenu.cs

```diff
@@ -13,114 +13,73 @@
  *  You should have received a copy of the GNU General Public License
  *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
  */
-
 using System.Text;
-using Microsoft.Extensions.Logging;
 
 namespace CounterStrikeSharp.API.Modules.Menu
 {
     public class CenterHtmlMenu : BaseMenu
     {
-        public CenterHtmlMenu(string title) : base(ModifyTitle(title))
-        {
-        }
-        
-        public override ChatMenuOption AddMenuOption(string display, Action<CCSPlayerController, ChatMenuOption> onSelect, bool disabled = false)
-        {
-            var option = new ChatMenuOption(ModifyOptionDisplay(display), disabled, onSelect);
-            MenuOptions.Add(option);
-            return option;
-        }
-        
-        private static string ModifyTitle(string title)
+        public CenterHtmlMenu(string title) : base(title)
         {
-            if (title.Length > 32)
```

Comment:
These checks were in place for a reason, why have you removed them?

Titles or menu items longer than these character limits cause it to wrap onto a new line and break the menu display.